pkg/server/fixtures/echo.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+echo -n `cat`
```

Comment:
Haven't looked at the tests yet (will do once first batch of comments is addressed), but maybe we want something less trivial than the identity function, to prove that we're not returning what we get passed in :)?

Reply:
Fixed.